docs/guide/writing-tasks.md

```diff
@@ -620,6 +620,7 @@ And some environment variables can be set to control behavior of the Cirrus CI A
 | CIRRUS_AGENT_VERSION                | not set                                                                                                                                                | Cirrus Agent version to use. If not set, the [latest release](https://github.com/cirruslabs/cirrus-ci-agent/releases)                                                                                                                                                                                                                                                                                               |
 | CIRRUS_AGENT_EXPOSE_SCRIPTS_OUTPUTS | not set                                                                                                                                                | If set, instructs Cirrus Agent to stream scripts outputs to the console as well as Cirrus API. Useful in case your Kubernetes cluster has logging collection enabled.                                                                                                                                                                                                                                               |
 | CIRRUS_CLONE_DEPTH                  | `0` which will reflect in a full clone of a single branch                                                                                              | Clone depth.                                                                                                                                                                                                                                                                                                                                                                                                        |
+| CIRRUS_FETCH_TAGS                   | `false` which will not fetch any tags                                                                                                                  | Whether to fetch Git tags.                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
```suggestion
| CIRRUS_CLONE_TAGS                   | `false` which will not fetch any tags                                                                                                                  | Whether to fetch Git tags.                                                                                                                                                                                                                                                                                                                                                                                                        |
```

And you need to add `CIRRUS_CLONE_TAGS` to https://github.com/cirruslabs/cirrus-ci-docs/blob/master/.spelling